### PR TITLE
Clarifying the template's intent

### PR DESCRIPTION
**Pull request by martinheidegger**
Clarifying the optional and required parts of the template to be listed as an event on the homepage and in the calendar.
Also making sure that organizers know how to use the template.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/meeting.md [hunk 1]
```diff
@@ -7,22 +7,40 @@ assignees: ''
...
+
+Change the meeting time below!
+
+Example: 04 Nov 2020, 13:40 Asia/Tokyo
+Valid Timezones: https://en.wikipedia.org/wiki/List_of_tz_database_time_zones
+-->
 
 :alarm_clock: Time: DD MMM YYYY, hh:mm GMT
 
-:link: https://zoom.us/j/181036238 (Native)
```
Comment (ninabreznik) [inline .github/ISSUE_TEMPLATE/meeting.md:16]: We could add



:link: https://zoom.us/j/181036238 (Native)<br/>
:link: https://zoom.us/wc/join/181036238?pwd= (Web)

**Comment by ninabreznik** [comment]
@martinheidegger I added some recommendations. If you think it's ok, we can go with that, otherwise also your suggestion was ok, so if you think it makes things to messy, we can leave out the suggested other links for now... I don't have super strong opinion about it.

**Comment by martinheidegger** [comment]
Merged the suggestions & PR, thanks for looking over it.

**Comment by ninabreznik** [comment]
Thank YOU for reacting to my suggestion and made the effort.

**Pull request merged**